Sources/XcodeGraph/Models/Metadata/XCFrameworkMetadata.swift

```diff
@@ -13,6 +13,7 @@ public struct XCFrameworkMetadata: Equatable {
     public var swiftModules: [AbsolutePath]
     /// All `.modulemap` files present in the `.xcframework`
     public var moduleMaps: [AbsolutePath]
+    public var expectedSignature: String?
```

Comment:
Why isn't this type?

```suggestion
    public var expectedSignature: XCFrameworkSignature?
```